manifests/03_credentials_request_powervs.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+   name: ibm-powervs-block-csi-driver-operator
+   namespace: openshift-cloud-credential-operator
+   annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+spec:
+  secretRef:
+     name: ibm-powervs-cloud-credentials
```

Comment:
changed the secret ref from `powervs-cloud-credentials` to `ibm-powervs-cloud-credentials` as that would be appropriate(this secret will be used by the `ibm-powervs-block-csi-driver-operator`).